addons/binding/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/handler/TeslaHandler.java

```diff
@@ -137,35 +137,42 @@ public void initialize() {
 
         logger.trace("Initializing the Tesla handler for {}", getThing().getUID());
 
+        updateStatus(ThingStatus.OFFLINE);
```

Comment:
@kaikreuzer Maybe, maybe not. As this point in the code, from the binding's point of view, it is certain that the Thing is OFFLINE as none of the threads to make the connection to the car have actually started.